src/module.rs

```diff
@@ -51,6 +52,38 @@ impl Module {
 
     /// Parse the LLVM bitcode (.bc) file at the given path to create a `Module`
     pub fn from_bc_path(path: impl AsRef<Path>) -> Result<Self, String> {
+        unsafe fn parse_bc(
+            context_ref: LLVMContextRef,
+            mem_buf: LLVMMemoryBufferRef,
+            out_module: *mut LLVMModuleRef,
+        ) -> LLVMBool {
+            let result =
+                llvm_sys::bit_reader::LLVMParseBitcodeInContext2(context_ref, mem_buf, out_module);
+            LLVMDisposeMemoryBuffer(mem_buf);
+            result
+        }
+        Self::from_path(path, parse_bc)
+    }
+
+    /// Parse the LLVM text IR (.ll) file at the given path to create a `Module`
+    pub fn from_ir_path(path: impl AsRef<Path>) -> Result<Self, String> {
+        unsafe fn parse_ir(
+            context_ref: LLVMContextRef,
+            mem_buf: LLVMMemoryBufferRef,
+            out_module: *mut LLVMModuleRef,
+        ) -> LLVMBool {
+            llvm_sys::ir_reader::LLVMParseIRInContext(context_ref, mem_buf, out_module, null_mut())
+        }
+        Self::from_path(path, parse_ir)
+    }
+    fn from_path(
+        path: impl AsRef<Path>,
+        parse: unsafe fn(
+            ContextRef: LLVMContextRef,
+            MemBuf: LLVMMemoryBufferRef,
+            OutModule: *mut LLVMModuleRef,
```

Comment:
Rust convention is `snake_case` for function arguments

Reply:
Will do. Should I amend the original commit or add a separate fixup?

Reply:
Either is fine.